data/migrations/20200609223613_event_ingredients.js

```diff
@@ -0,0 +1,14 @@
+
+exports.up = function(knex) {
+  return knex.schema.createTable("Event_Ingredients", tbl => {
+    tbl.increments();
+    tbl.integer('event_id').notNullable().unsigned().references('Events.id');
+    tbl.string('description', 255).notNullable();
+    tbl.boolean('requested').notNullable().defaultTo(false);
+    tbl.integer('user_id').notNullable().unsigned().references('Users.id');
```

Comment:
This needs to be nullable so it can be blank if `requested=true`.

Reply:
Resolved in commit https://github.com/Lambda-School-Labs/neighborhood-chef-be/pull/92/commits/78ba4c52ede873f16cfe70e393daa932d6eaebfd